app/uk/gov/hmrc/brm/utils/BrmException.scala

```diff
@@ -41,12 +41,18 @@ trait BrmException extends Controller {
   }
 
   def notFoundPF(message: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+    case Upstream4xxResponse(m, FORBIDDEN, _, _) if(m.contains("BIRTH_REGISTRATION_NOT_FOUND")) =>
+      logException(Some(message), Some("Forbidden returned from NRS for not found"), NOT_FOUND)
+      Ok(Json.toJson(BirthResponseBuilder.withNoMatch()))
     case Upstream4xxResponse(m, NOT_FOUND, _, _) =>
-      logException(Some(message), Some("notFound"), NOT_FOUND)
+      logException(Some(message), Some("NotFound"), NOT_FOUND)
       Ok(Json.toJson(BirthResponseBuilder.withNoMatch()))
   }
 
   def badRequestPF(message: String)(implicit payload: Payload) : PartialFunction[Throwable, Result] = {
+    case Upstream4xxResponse(m, FORBIDDEN, _, _) if(m.contains("INVALID_DISTRICT_NUMBER")) =>
```

Comment:
@chris-ianson @manishsw what happens if this does not contain BIRTH_REGISTRATION_NOT_FOUND?